keras/src/random/random_test.py

```diff
@@ -380,12 +381,11 @@ def test_uniform_dtype_validation(self):
         reason="This test requires `jax` as the backend.",
     )
     def test_dropout_jax_jit_stateless(self):
-        import jax
         import jax.numpy as jnp
 
         x = ops.ones(3)
 
-        @jax.jit
+        @nnx.jit
```

Comment:
Would the integration prevent the use of `jax.jit` with Keras layers?

Reply:
yes! it would only work with nnx.jit for now ( They might be working on adding support for jax.jit)

Reply:
Added nnx as a opt in with this flag - os.environ["KERAS_NNX_ENABLED"]